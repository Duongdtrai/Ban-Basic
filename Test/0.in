44
14
18 31 95 100 76 65 87 92 42 9 85 47 37 80 
45
15 84 29 54 27 100 24 4 47 27 55 1 11 59 34 81 2 77 83 20 88 38 88 78 3 98 27 50 92 2 45 77 12 60 68 42 35 24 66 18 1 57 62 23 1 
87
48 75 95 32 74 95 62 27 47 91 70 47 24 100 86 84 22 34 67 78 14 44 62 6 26 64 78 71 66 73 27 18 33 29 4 62 25 66 61 91 45 89 1 93 90 57 85 78 51 23 92 38 93 59 5 60 42 18 100 12 13 82 27 45 44 1 5 56 40 3 72 10 98 68 100 89 43 9 3 7 32 96 47 98 22 33 46 
65
65 63 58 13 15 94 73 18 65 6 89 23 99 57 55 2 33 11 73 48 7 2 31 70 63 40 83 1 25 12 87 14 2 32 89 42 66 47 89 49 34 31 30 62 19 7 72 60 4 42 62 79 21 51 53 78 92 28 21 63 85 60 19 53 75 
99
39 4 56 58 12 11 40 20 2 32 31 36 37 77 97 22 93 53 94 94 91 14 9 36 62 27 10 2 96 27 67 66 85 31 9 21 92 16 19 81 42 43 97 34 100 70 34 26 10 93 35 9 47 40 5 85 82 81 1 13 13 54 97 48 15 59 77 16 69 17 47 19 15 74 75 9 59 39 80 35 64 95 66 54 60 40 61 1 6 10 69 79 24 88 25 16 31 52 79 
82
60 5 69 27 46 24 100 58 29 14 72 87 64 88 90 17 50 50 53 6 90 73 55 8 56 58 63 71 28 3 95 74 20 77 28 81 53 86 57 14 28 49 42 89 82 31 44 5 33 39 54 48 84 72 100 87 45 88 92 13 28 27 82 21 70 52 38 19 12 46 45 58 91 10 72 40 31 71 96 69 88 37 
73
90 4 92 90 100 27 2 17 50 92 26 99 99 91 85 33 89 25 66 72 75 7 93 51 70 68 1 84 46 41 15 90 18 89 73 46 30 81 46 80 54 9 19 24 40 33 30 96 33 15 74 89 85 84 88 58 55 94 18 43 93 71 84 67 77 31 34 20 96 36 19 100 95 
80
43 52 29 13 70 49 15 43 16 62 42 36 45 12 55 24 90 43 62 29 98 36 64 95 17 96 86 62 44 78 35 82 29 81 75 37 75 51 77 7 68 54 42 2 56 97 75 27 32 88 83 19 75 18 78 18 48 50 19 36 89 19 93 26 60 59 66 71 98 87 47 76 2 81 24 67 33 49 96 90 
94
76 93 92 9 95 84 10 4 42 84 22 19 23 52 47 26 45 13 58 44 23 45 58 84 38 68 17 84 44 15 1 53 49 34 76 59 32 19 19 72 12 37 99 80 33 27 30 12 80 57 42 10 9 75 61 74 82 4 24 75 42 55 19 34 63 44 79 13 83 30 20 57 46 5 51 96 63 72 72 78 68 88 40 64 15 27 15 78 36 87 98 51 96 37 
98
4 19 66 94 34 20 61 11 36 60 79 80 45 87 62 84 90 49 94 84 39 92 67 97 44 23 95 33 50 100 92 37 87 48 30 28 47 83 76 78 81 29 94 81 11 71 39 64 7 75 9 43 9 8 12 67 10 36 95 58 91 45 61 22 21 61 70 8 95 26 15 65 29 1 75 56 54 60 74 10 24 3 61 43 10 61 89 81 62 84 42 100 39 70 82 43 10 95 
77
75 51 11 32 89 53 86 44 15 85 65 32 60 87 19 10 31 43 93 55 9 82 51 23 30 45 90 18 43 34 31 41 53 29 64 18 22 69 73 21 17 68 20 99 31 14 29 70 5 4 36 87 66 80 26 43 65 48 8 36 32 11 34 89 76 87 77 92 88 79 30 65 42 62 99 87 42 
75
62 48 20 37 15 8 50 66 31 93 74 19 79 56 58 2 99 98 3 11 31 96 65 21 40 65 24 2 45 75 53 25 94 36 61 3 33 20 85 36 71 86 94 74 28 16 68 74 77 4 56 51 65 1 45 31 82 24 11 99 3 69 59 17 20 48 97 71 16 68 36 68 12 59 23 
35
75 43 87 9 73 43 16 57 99 40 53 94 93 58 45 47 66 11 94 89 16 81 32 22 70 17 51 53 53 52 86 33 48 88 78 
73
15 74 93 88 52 22 93 18 15 94 99 22 58 43 31 65 58 67 6 94 54 68 11 18 21 34 66 27 10 89 99 30 85 55 59 74 86 5 14 91 2 22 78 49 67 16 100 63 41 26 96 40 72 83 75 24 68 55 99 13 53 77 53 51 59 72 66 20 30 69 24 41 41 
52
46 21 44 64 4 89 51 18 55 45 77 71 61 20 39 15 26 26 43 80 80 25 69 81 77 11 12 77 61 86 97 1 86 29 99 8 14 58 35 100 2 26 40 36 88 86 94 17 47 11 64 11 
63
36 100 46 40 35 41 15 38 86 88 46 66 98 30 97 51 45 42 88 84 79 87 25 87 66 24 95 63 95 76 13 90 14 40 48 67 13 23 30 89 7 23 94 50 72 19 68 20 91 98 54 51 5 25 24 44 69 60 46 89 42 65 26 
39
46 66 57 60 26 96 80 53 78 4 33 71 51 68 40 20 79 53 22 73 82 11 30 12 71 1 5 34 53 97 90 8 89 97 21 94 92 62 43 
78
55 28 19 1 30 32 23 61 54 8 88 81 62 77 11 99 88 6 13 6 69 13 70 28 47 80 60 50 39 33 93 54 64 87 52 41 13 63 85 32 30 9 16 63 100 68 79 52 86 74 86 89 70 24 69 59 49 2 29 98 68 48 69 16 10 93 80 6 82 15 82 76 38 55 41 50 28 2 
10
27 34 65 80 77 34 28 73 30 7 
16
44 93 94 20 20 84 91 33 82 73 23 34 59 78 97 57 
28
27 54 63 58 64 86 18 96 7 91 36 13 85 12 46 54 27 60 83 31 41 12 9 56 29 33 15 49 
61
75 64 73 16 3 71 67 15 92 30 70 63 56 83 40 64 49 1 12 74 90 34 100 33 78 49 28 8 48 21 97 89 99 15 45 4 38 29 91 29 19 56 79 39 10 65 61 90 2 52 49 53 9 11 89 37 2 36 24 56 57 
12
45 31 10 81 40 6 52 59 59 30 5 17 
83
13 33 87 86 20 42 78 16 70 75 13 55 44 50 46 25 41 53 43 83 34 81 79 47 22 65 72 39 86 96 6 10 69 84 43 23 60 67 20 21 15 84 55 67 95 79 4 65 31 82 91 90 10 31 28 53 39 53 31 26 50 94 34 29 64 94 58 94 4 88 65 47 99 80 92 27 80 60 37 37 7 5 89 
11
72 82 31 10 41 60 9 62 64 18 59 
94
79 69 60 77 4 82 41 12 84 81 43 51 57 26 71 42 55 42 27 23 31 60 45 41 36 98 85 22 84 57 67 72 87 23 10 45 44 85 92 8 46 52 93 67 80 69 39 2 14 1 21 58 58 98 82 61 54 57 4 63 32 99 7 23 13 23 68 7 36 32 52 14 43 96 12 42 20 17 17 19 48 16 33 91 53 54 53 88 47 76 25 61 34 1 
99
87 46 87 73 62 71 5 64 33 80 73 1 4 30 100 38 89 29 23 27 35 49 25 86 35 78 55 21 74 2 77 35 85 83 54 74 15 31 50 83 26 54 40 84 94 29 44 70 79 78 63 39 72 10 33 42 34 90 43 77 49 3 82 82 5 12 1 71 34 7 94 2 65 90 74 49 74 63 43 65 59 87 48 50 26 59 44 41 49 64 28 25 60 70 88 20 84 53 95 
39
6 93 40 23 7 51 27 64 47 45 6 24 1 61 87 38 65 54 54 40 40 67 14 2 20 21 48 65 28 20 93 5 87 26 43 28 6 19 7 
17
69 3 17 75 1 61 23 71 36 40 58 94 44 23 91 71 96 
1
72 
3
48 84 24 
13
28 7 71 16 89 23 91 69 17 40 35 57 85 
60
41 16 98 74 92 93 90 37 80 22 76 16 97 85 35 16 30 85 96 24 24 79 43 38 81 99 74 28 81 54 69 37 1 56 95 93 14 29 88 52 81 41 100 3 37 78 4 78 14 82 22 11 53 34 47 39 90 17 1 3 
14
41 47 55 53 88 76 7 76 52 50 52 36 97 93 
93
61 10 32 88 49 99 79 13 44 27 2 68 26 53 62 42 45 79 80 65 70 69 53 67 28 69 14 57 11 21 6 53 14 50 38 27 93 52 98 73 22 39 37 45 81 86 24 92 34 78 40 9 36 28 34 81 67 66 15 85 95 69 72 97 15 31 60 9 84 9 92 79 5 51 61 51 87 67 46 87 80 77 92 39 100 7 96 7 13 62 96 30 63 
80
93 38 7 60 40 8 9 95 24 28 96 19 59 76 55 95 32 67 42 1 64 67 10 95 45 55 50 4 81 44 47 93 64 67 60 21 38 71 67 43 78 57 19 60 32 87 97 11 54 25 42 97 39 77 45 37 55 47 5 88 84 48 100 50 84 64 99 90 71 7 97 8 88 20 79 74 72 21 4 37 
77
47 52 46 45 97 32 16 40 21 85 60 65 63 96 31 59 86 96 52 42 34 81 79 46 16 21 32 44 67 54 29 16 45 3 69 42 52 21 36 73 26 99 98 74 31 19 83 62 29 96 76 87 80 69 15 79 15 37 49 44 93 87 19 6 26 71 52 58 29 15 37 7 55 83 14 82 62 
9
50 52 39 13 73 21 36 38 25 
3
71 72 80 
16
36 74 3 57 100 55 9 39 46 1 35 29 40 15 69 64 
81
14 32 59 42 76 55 22 48 13 93 37 36 3 43 22 64 51 71 58 42 87 26 73 57 82 35 63 42 17 99 70 6 14 53 18 5 2 57 17 64 90 23 99 54 52 20 12 66 31 37 95 24 13 25 47 30 48 13 29 48 86 76 76 59 72 10 63 78 32 50 25 2 77 54 47 38 25 57 10 97 72 
50
84 39 39 16 57 11 31 75 38 23 37 74 1 59 20 17 64 100 18 52 88 18 28 60 39 95 35 98 48 28 28 41 76 88 54 35 65 21 44 54 83 79 15 63 47 9 45 62 21 51 
7
13 88 9 32 83 42 59 
23
52 31 80 55 12 67 23 18 36 91 43 33 38 4 22 57 53 27 15 32 57 7 55 
