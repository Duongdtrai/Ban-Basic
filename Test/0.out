3886974
229073018
8310450978
151614096
12866165172
3884608092
5776172526
3933678522
1016096040
15317108752
5720582448
4335007732
92758860
4722621540
510733900
1310794576
199056710
859313490
430614
6974632
11292600
1718653454
595218
1384826400
696528
9545446170
16391940922
87378112
8299508
0
336
418200
1777442604
8262492
4580531046
945101256
32702166
60522
1440
1493056
636216120
439207172
138688
2621520
